articles/express/index.md

```diff
@@ -0,0 +1,318 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/express/
+title: Introduction to Express.js
+description: Express is a lightweight framework on top of Node.js, it adds functionality like middleware, template engines, and routing.
+author: rohan-reddy
+date: 2020-09-18T00:00:00-11:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/express/hero.jpg
+    alt: expressjs example text image
+---
+[Node.js](https://www.section.io/engineering-education/history-of-nodejs/) is a JavaScript run time environment which is used to create server-side applications and tools. Node.js is fast, portable and written in JavaScript. Node.js does not directly support common tasks such as handling requests, serving files, handling HTTP methods such as `GET` and `POST` this is where Node.js's [rich ecosystem](https://www.section.io/engineering-education/most-useful-nodejs-packages/) comes to our aid.
```

Comment:
Add but before this

Reply:
@rohanreddych, you still need to modify this. 
Please add the word but where I've specified so the sentence makes sense.

Also, you should replace the last comma in the list with and.

Reply:
>Node.js is fast, portable and written in JavaScript **but** Node.js does not directly support comm ... 
is this correct?